domains/krish.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "krish858",
+           "email": "unknowmortal57@gmail.com",
+           "discord": "789102469345968131"
+        },
+    
+        "record": {
+            "CNAME": "krish858.github.io"
```

Comment:
404 page, please give a preview of your website

Reply:
Preview: krish858.github.io/portfolio

Reply:
> 404 page, please give a preview of your website

preview:%20krish858.github.io/portfolio

Reply:
Complete your website

Reply:
What 21Z said.